eslzArm/eslzArm.json

```diff
@@ -1192,6 +1263,161 @@
                 }
             }
         },
+        {
+            // Deploying ESLZ monitor custom policies. Note: all policies should eventually be moved to built-in policies and codebase will be reduced
```

Comment:
is this comment correct? especially the built-in comments?

At a minimum lets call out its the AMBA policies in the comment to make it easier to troubleshoot

Reply:
Werrlll I suspect we always want to go to built-in policies, but admittedly the timeline is not there yet
I'll update the comment. with AMBA naming